helpers/scripts/copy_prod_data.js

```diff
@@ -0,0 +1,77 @@
+var PROD_ADDRESS = 'http://api.grebelife.com'
```

Comment:
This script will break with https://github.com/CGUC/skybunk-server/pull/71/files just fyi - so depending on which PR ships first some changes will have to be made to the other.

Reply:
Updated